Editor/Scripts/CodeGenerators/RGActionCodeGenerator.cs

```diff
@@ -164,6 +164,8 @@ private static string SearchForBotStates()
                     string className = classDeclaration.Identifier.ValueText;
                     List<RGStateInfo> stateList = new List<RGStateInfo>();
 
+                    string nameSpace = classDeclaration.Ancestors().OfType<NamespaceDeclarationSyntax>().FirstOrDefault()?.Name.ToString();
```

Comment:
can you update this higher in the file in the other location so that the code pattern matches

~line 75

Reply:
Updated to be the first line of the for loop. Is that what you meant? If not let me know 👍

Reply:
no, I mean on line ~75 where this similar operation occurs, can you update that other code location to be consistent with this one

Reply:
I updated the declaration to match. Let me know if that's more like what your saying